packages/appium/lib/cli/extension.js

```diff
@@ -30,7 +35,7 @@ async function runExtensionCommand (args, configObject) {
   const logFn = (msg) => log(json, msg);
   let config = configObject;
   config.log = logFn;
-  const CommandClass = type === DRIVER_TYPE ? DriverCommand : PluginCommand;
+  const CommandClass = /** @type {ExtType extends import('../extension/manifest').DriverType ? import('@appium/types').Class<DriverCommand> : import('@appium/types').Class<PluginCommand>} */(commandClasses[type]);
```

Comment:
can we put this in a typedef or something so it doesn't impact readability so much?